### ISSUE TITLE
falling loop-block

### ISSUE THREAD

**Issue by OverBrave** [SkriptLang/Skript#4560]
### Skript/Server Version

```
```
[22:20:18 INFO]: [Skript] Skript's aliases can be found here: https://github.com/SkriptLang/skript-aliases
[22:20:18 INFO]: [Skript] Skript's documentation can be found here: https://skriptlang.github.io/Skript
[22:20:18 INFO]: [Skript] Server Version: git-Pufferfish-39 (MC: 1.18.1)
[22:20:18 INFO]: [Skript] Skript Version: 2.6.1
[22:20:18 INFO]: [Skript] Installed Skript Addons:
[22:20:18 INFO]: [Skript]  - DiSky v3.6.0 (https://disky.itsthesky.info/)
[22:20:18 INFO]: [Skript]  - skript-yaml v1.4
[22:20:18 INFO]: [Skript]  - skript-reflect v2.2.3 (https://github.com/TPGamesNL/skript-reflect)
[22:20:18 INFO]: [Skript]  - SharpSK v1.6.13
[22:20:18 INFO]: [Skript] Installed dependencies:
[22:20:18 INFO]: [Skript]  - WorldGuard v7.0.6+1cff55f
```
```

### Bug Description

The line that worked in 2.6 no longer works in 2.6.1.

### Expected Behavior

It should detect loop-blocks.

### Steps to Reproduce

This is just example, it doesn't detect loop-block.

```
		loop blocks in radius 5 around player's location:
			spawn 1 of falling loop-block at loop-block
```
or
```
		loop blocks in radius 5 around player's location:
			spawn 1 of falling block of loop-block at loop-block
```

### Errors or Screenshots

```
There's no loop that matches 'loop-block at loop-block' (scsmash.sk, line 124: spawn 1 of falling loop-block at loop-block')
There's no loop that matches 'loop-block at loop-block' (scsmash.sk, line 140: spawn 1 of falling block of loop-block at loop-block')
```

### Agreement

- [X] I have read the guidelines above and confirm I am following them with this report.

**Comment by TPGamesNL** [comment]
That syntax (falling blocks of variable type) doesn't exist in Skript, neither 2.6 nor 2.6.1. Perhaps it was an addon that you removed?

**Comment by Jjman739** [comment]
Yeah, I never understood why it works for any one block, but not variables. Would be a nice addition, but it's not a bug.

**Comment by AyhamAl-Ali** [comment]
That's aliases feature if I'm not mistaken, but this can be an suggestion to add `falling block of %itemtype%`

**Comment by OverBrave** [comment]
It's weird, I don't understand. All I did was upgrade from 2.6 to 2.6.1. I could swear it works on version 2.6.

**Comment by Jjman739** [comment]
What addons do you have? Maybe one of them isn't compatible with the new version.

**Comment by AyhamAl-Ali** [comment]
As TP stated, you probably had Skirt addon that has this [expression](https://skripthub.net/docs/?id=6588)

**Comment by OverBrave** [comment]
Honestly, this is the first time I've heard of this addon. To be honest I was using SkQuery. Could this be broken in the latest 1.18.1 release? I don't think it has that though.

**Comment by TPGamesNL** [comment]
In that case it was probably SkQuery: https://docs.skunity.com/syntax/search/id:1287

I'm reopening this as a suggestion to make an expression for this syntax to allow for non-literal usage.

**Comment by OverBrave** [comment]
Anyway, it's okay. I handled it with SkReflect. But I think it would be nice if this feature could be brought.

**Comment by OverBrave** [comment]
Thanks, it would be nice to add this expression.

**Comment by UsainSrht** [comment]
You can use something like
`!spawn ("falling %type of target block%" parsed as entitytype) at player`
till it's added.
